custom-connectors/Public360/Artifacts/apiDefinition.swagger.json

```diff
@@ -0,0 +1,281 @@
+                    },
+                    "Notes": {
+                      "type": "string",
+                      "description": "Notes",
+                      "title": ""
+                    }
+                  },
+                  "description": "parameter",
+                  "required": [
+                    "Title",
+                    "Category",
+                    "Status",
+                    "CaseNumber"
+                  ]
+                }
+              },
+              "required": [
+                "parameter"
+              ]
+            }
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        }
+      }
+    },
+    "/Biz/v2/api/call/SI.Data.RPC/SI.Data.RPC/CaseService/CreateCase": {
+      "post": {
+        "summary": "Create Case",
+        "description": "Create Case",
+        "operationId": "CreateCase",
+        "parameters": [
+          {
+            "name": "hosturl",
+            "in": "query",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "parameter": {
+                  "type": "object",
+                  "properties": {
+                    "Title": {
+                      "type": "string",
+                      "description": "Title",
+                      "title": ""
+                    },
+                    "CaseType": {
+                      "type": "string",
+                      "description": "Case Type",
+                      "title": "Case Type"
+                    },
+                    "ResponsiblePersonRecno": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "Responsible Person Recno",
+                      "title": "Responsible Person Recno"
+                    },
+                    "ResponsibleEnterpriseRecno": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "Responsible Enterprise Recno",
+                      "title": "Responsible Enterprise Recno"
+                    },
+                    "ProgressPlanId": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "Progress Plan Id",
+                      "title": "Progress Plan Id"
+                    }
+                  },
+                  "description": "parameter",
+                  "required": [
+                    "Title"
+                  ]
+                }
+              },
+              "required": [
+                "parameter"
+              ]
+            }
+          }
+        ],
+        "responses": {
+          "default": {

```

Comment:
@Pranav2889 Please replace the `default` responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.